src/frontend/src/lib/types/onramper.ts

```diff
@@ -1,3 +1,7 @@
 // The list of networks that are supported by OnRamper can be found here:
 // https://docs.onramper.com/docs/network-support
 export type OnRamperNetworkId = 'icp' | 'bitcoin' | 'ethereum';
+
+// The list of cryptocurrencies that are supported by Onramper can be found here:
+// https://docs.onramper.com/docs/crypto-asset-support
+export type OnramperCryptoId = string;
```

Comment:
What about just `OnramperId` given that's the term used in the linked documentation?

Reply:
We will have a `OnRamperFiatId` too, I wanted to distinguish, even if they are both strings in the end, WDYT?

Reply:
Honestly? When I open the link provided in the comment, https://docs.onramper.com/docs/crypto-asset-support, I started looking for the keyword "crypto" to find the ID

Reply:
ah! yes.. make sense, changed!